lib/src/util/react_wrappers.dart

```diff
@@ -363,24 +403,35 @@ CallbackRef chainRef(ReactElement element, CallbackRef newCallbackRef) {
         'The existing ref is a String ref and cannot be chained');
   }
 
-  if (existingRef is! Function) {
```

Comment:
[Whitespace-agnostic diff](https://github.com/Workiva/over_react/pull/338/files?w=1) works better for viewing the changes in this method